project/redis-master.sls

```diff
@@ -0,0 +1,17 @@
+# Install Redis, listening on all interfaces.
+# This does not open the firewall for Redis, so do that elsewhere
```

Comment:
I don't know; I just copied this from hosted-tracpro.  Maybe it just makes it easier to use it from other servers later on if needed?